js/app.js

```diff
@@ -0,0 +1,186 @@
+'use strict';
+/*
+Objectives:
+Create a separate JS object literal for easch shop location that does the following:
+1. Store the min/max hourly customers, and the average cookies per customer, in object properties
+
+2. Use a method of that object to generate a random number of customers per hour. (Math.random())
+
+3. Calculate and store the simulated amounts of cookies purchased for each hour at each location using average cookies purchased and the random number of customers generated
+
+4. Store the results for each location in a separate array... perhaps as a property of the object representing that location
+
+5. Display the values of each array as unordered lists in the browser
+
+6. Calculate the sum of these hourly totals
+*/
+
+/*
+References:
+1. Min/Max inclusive random number generator:
+https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+*/
+
+// stores
+
+var firstAndPike = {
+  name: '1st and Pike',
+  minCust: 23,
+  maxCust: 65,
+  aveCust: 6.3,
+  openTime: 6,
+  closeTime: 20,
+  salesArray: [],
+  randomCust: function(){
+    var min = Math.ceil(this.minCust);
```

Comment:
It seems that we already expect minCust and maxCust to be integers, so Math.floor() and Math.ceil() are not needed here. Even if we allow non-integer values for these, the math.foor() call you have when we use them with the random number generation means these are extra and unneeded.